academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,170 @@
+
+</HighlightBox>
+
+Although application-specific blockchains offer superior scalability compared to general-purpose blockchain platforms, smart contract development for general-purpose chains and generic virtual machines (VMs) like in Ethereum offer their own benefits. IBC provides a method of incorporating the strengths of general-purpose and application-specific blockchains into unified overall designs. For example, it allows a Cosmos chain tailored towards performance and scalability to use funds that originate on Ethereum and possibly record events in a Corda distributed ledger or, in the reverse, a Corda ledger initiating the transfer of underlying assets defined in Cosmos or Ethereum.
+
+![Internet of blockchains](images/internetofchains.png)
+
+With cross-chain communication via IBC, a decentralized network of independent and interoperable chains exchanging information and assets is possible. This "internet of blockchains" brings the promise of increased and seamless scalability. In Cosmos, the vision being implemented is to have a universe of independent chains that are all connected using peg-zones as bridges between the Cosmos network and chains outside of it and connecting all chains via hubs. All of these make up the internet of blockchains.
+
+## Interchain Standards (ICS)
+
+There are two categories (aside from __meta__) that define ICS in the [general ICS definition](https://github.com/cosmos/ibc/blob/master/spec/ics-001-ics-standard/README.md):
+
+* **IBC/TAO:** a standard defining an inter-blockchain communication system core transport, authentication, and ordering-layer protocol.
+* **IBC/APP:** a standard defining an inter-blockchain communication system application-layer protocol.
+
+Assume we have two chains: **chain *A* and chain *B***. Both will implement the ICS/TAO for IBC.
+
+![IBC overview - two connected chains](images/ibcoverview.png)
+
+Notice three crucial elements in the diagram:
+
+* The chains depend on relayers to communicate.
+* Many relayers can serve one or more channels to send messages between the chains.
+* Each side of the relay uses the light client of the other chain to quickly verify incoming messages.
+
+### IBC/TAO - transport layer
+
+The transport layer (TAO) provides the necessary infrastructure to establish secure connections and authenticate data packets between chains. The application layer defines exactly how data packets should be packaged and interpreted by the sending and receiving chains.
+
+The great promise of IBC is the possibility of optimizing not only on the transport layer but also on the layers above it: a powerful and secure but ultimately generic and permissionless transport layer, which supports any number of diverse and innovative applications from asset transfers and oracle data provision to decentralized identities and cross-chain, synchronous transaction validation built on top of it.
```

Comment:
"The great promise of IBC is the possibility of optimizing not only on the transport layer but also on the layers above it"
Is it really the optimization that is the big draw? I would point more to providing a reliable, permissionless and generic base layer while allowing for composability/modularity with separation of concerns by moving application stuff to a higher level layer. Thoughts @charleenfei ?

Reply:
I think I recognize this sentence from my blogpost haha. I think your sentence and the blogpost sentence have the same thing, but as this is a developer tutorial and not a promo blogpost for Interchain Accounts would make sense to have a less....dramatic sentence probably. "providing a reliable, permissionless and generic base layer while allowing for composability/modularity with separation of concerns by moving application stuff to a higher level layer." sounds good to me!